File: metric_tool/test/resources/dart/wmc/wmc_j.dart
library mylib;

class WmcJ {
  void method1() {
    switch (0) {
      case 0:
        break;
      case 1:
        break;
      case 2:
        switch (2) {
          case 0:
            break;
          default:
            break;
        }
        break;
      default:
        break;
    }
  }

  void method2() {
    switch (0) {
      case 0:
        break;
      default:
        break;
    }
  }
}

File: metric_tool/test/resources/kotlin/wmc/WmcJ.kt
package io.company.app.view.basket.kotlin.wmc

internal class WmcJ {
    fun method1() {
        when (0) {
            0 -> Unit
            1 -> Unit
            2 -> when (2) {
                0 -> Unit
                else -> Unit
            }
            else -> Unit
        }
    }

    fun method2() {
        when (0) {
            0 -> Unit
            else -> Unit
        }
    }
}
